metadata_service/proxy/neo4j_proxy.py

```diff
@@ -670,11 +773,11 @@ def get_tags(self) -> List:
         :return:
         """
         LOGGER.info('Get all the tags')
-        # todo: Currently all the tags are default type, we could open it up if we want to include badge
+        # all the tags are default type
         query = textwrap.dedent("""
         MATCH (t:Tag{tag_type: 'default'})
-        OPTIONAL MATCH (resource)-[:TAGGED_BY]->(t)
-        RETURN t as tag_name, count(distinct resource.key) as tag_count
+        OPTIONAL MATCH (tbl:Table)-[:TAGGED_BY]->(t)
```

Comment:
this is not correct...https://github.com/amundsen-io/amundsenmetadatalibrary/pull/175